src/natcap/invest/hra.py

```diff
@@ -1782,10 +1782,13 @@ def _open_table_as_dataframe(table_path, **kwargs):
     if extension in {'.xls', '.xlsx'}:
         excel_df = pandas.read_excel(table_path, **kwargs)
         excel_df.columns = excel_df.columns.str.lower()
+        excel_df['path'] = excel_df['path'].apply(
```

Comment:
This makes me wonder again do we really need to support Excel files just for HRA?

Reply:
Fair question and I expect a carry over to ensure original functionality. I think this question could be relevant in the larger conversation of whether we should be stricter about our api contracts.